### PR TITLE
🔬  Section 8: Advanced Functions

### PR DESCRIPTION
**Pull request by Palwisha-18**

### CODE REVIEW

File: 08_Advanced_Functions/advanced_functions.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+const say = function (message) {
+  console.log(message);
+  console.log(this.type + ' says ' + this.sound);
+};
+
+say.apply(cat, ['What does a cat say?']); // this object in the say() function references the cat object.
+say.apply(dog, ['What does a dog say?']); // this object in the say() function references the dog object.
+
+/* What does a cat sound?
```
Comment (ahmadjanan) [inline 08_Advanced_Functions/advanced_functions.js]: sound -> say

File: 08_Advanced_Functions/advanced_functions.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+Cat says Meow
+What does a dog sound?
```
Comment (ahmadjanan) [inline 08_Advanced_Functions/advanced_functions.js]: sound -> say

**Review by ahmadjanan — CHANGES_REQUESTED** [review]

File: 08_Advanced_Functions/advanced_functions.js (hunk 1 above)
Comment (Palwisha-18) [reply 08_Advanced_Functions/advanced_functions.js]: fixed.

File: 08_Advanced_Functions/advanced_functions.js (hunk 2 above)
Comment (Palwisha-18) [reply 08_Advanced_Functions/advanced_functions.js]: fixed.

**Review by ahmadjanan — APPROVED** [review]
LGTM 👍

**Pull request merged**